.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will test a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
```

Comment:
This is the name of the job, so if you want it to show up as `test`, you can change it here.